### PR TITLE
HIVE-26726: Tinyint column with windowing fn crashes at runtime

### PR DESCRIPTION
**Pull request by scarlin-cloudera**
Added handling for a tinyint.

### What changes were proposed in this pull request?

Added tinyint handling for a windowng function

### Why are the changes needed?

To get the tinyint processing to work

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

Added qtest that failed before the change

### CODE REVIEW

File: ql/src/java/org/apache/hadoop/hive/ql/udf/ptf/ValueBoundaryScanner.java [hunk 1]
```diff
@@ -760,6 +760,7 @@ public static SingleValueBoundaryScanner getBoundaryScanner(BoundaryDef start, B
     case "int":
     case "bigint":
     case "smallint":
+    case "tinyint":
```
Comment (amansinha100) [inline ql/src/java/org/apache/hadoop/hive/ql/udf/ptf/ValueBoundaryScanner.java:763]: Good catch.  But can you also check if this type needs to be handled anywhere else for the window function code ? e.g line 730 above.

File: ql/src/test/queries/clientpositive/windowing_tinyint.q [hunk 2]
```diff
@@ -0,0 +1,11 @@
+--! qt:dataset:src
```
Comment (kasakrisz) [inline ql/src/test/queries/clientpositive/windowing_tinyint.q:1]: Please remove this line since this test doesn't use the `src` table

**Review by kasakrisz — CHANGES_REQUESTED** [review]

File: ql/src/test/queries/clientpositive/windowing_tinyint.q (hunk 2 above)
Comment (scarlin-cloudera) [reply ql/src/test/queries/clientpositive/windowing_tinyint.q]: Done

**Review by amansinha100 — APPROVED** [review]
+1

**Pull request merged**